contracts/src/CrossChainApplications/NativeTokenBridge/tests/NativeTokenSourceTests.t.sol

```diff
@@ -303,6 +306,43 @@ contract NativeTokenSourceTest is Test {
         );
     }
 
+    function _initMockTeleporterRegistry() internal {
```

Comment:
Can we consolidate this function that's repeated in multiple files?

Reply:
Michael's PR https://github.com/ava-labs/teleporter/pull/203 extracts some of the common functionality here. I'll make a followon PR to extract everything I can when his is all merged in.

Reply:
Sounds good!